registry.json

```diff
@@ -5419,5 +5419,19 @@
    "telegram": "Dinoaus73",
    "operatorName": "John Dean",
    "github": ""
+  },
+  {
+    "ticker": "FFR",
+    "poolId": "213ccec05701bd77a8613e48f09ea42504c50d5d709ad16339ef0638",
+    "social": {
+      "website": "https://fatfrog.io/",
+      "twitter": "FatFrog33546964",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "Dr K",
```

Comment:
You must provide a telegram handle and it must be that of an individual and not a group or channel. This looks to be a screen name rather than the username

Reply:
Sorry - you're right - changed to Fatfrog_FFR